drivers/main-device.js

```diff
@@ -229,7 +236,8 @@ module.exports = class mainDevice extends Homey.Device {
             const blower0 = await this.getComponent('BLOWER', components, '0');
             const blower1 = await this.getComponent('BLOWER', components, '1');
             const blower2 = await this.getComponent('BLOWER', components, '2');
-
+            const circulationPump = await this.getComponent('CIRCULATION_PUMP', components);
+console.log(circulationPump)
```

Comment:
Assuming this is only for development and in final would go away.

Reply:
hehe yes